modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/skin/Utils.java

```diff
@@ -693,6 +694,10 @@ public static void addMnemonics(ContextMenu popup, Scene scene) {
     }
 
     public static void addMnemonics(ContextMenu popup, Scene scene, boolean initialState) {
+        addMnemonics(popup, scene, initialState, null);
+    }
+
+    public static void addMnemonics(ContextMenu popup, Scene scene, boolean initialState, List<Mnemonic> into) {
```

Comment:
This method must not be public (or protected), or it becomes part of the public API.